_tutorials/development.md

```diff
@@ -56,8 +94,20 @@ In summary:
 1. Install the `gh` tool using `sudo apt update && sudo apt install gh`.
 2. Run `gh auth login` to authenticate.
 
+#### Option 3: Using a Personal Access Token
```

Comment:
Perhaps hyperlink to https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on

Reply:
Updated